deploy/template.yaml

```diff
@@ -1299,6 +1320,155 @@ Resources:
               Bool:
                 "aws:SecureTransport": true
 
+  #
+  # Metrics and Alarms
+  #
+
+  # API Gateway Alarms
+
+  # 4xx Status Code
+  ApiGateway4xxStatusAlarm:
+    Condition: IsNotDevelopment
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName: !Sub "${AWS::StackName}-ApiGateway4xxStatusAlarm"
+      AlarmDescription: "Trigger the alarm if errorThreshold exceeds 1% with the minimum of 10 invocations in 2 out of the last 5 minutes"
```

Comment:
This sounds great to me. I might just check in with @munhitsu, I know we've some some SLAs around errors (unless you've seen them?).

I think this is a much lower threshold than what we agreed (which is great! would rather know before we breach an SLA), but i might want to check those time windows... I've got 10 mins in my mind for somre reason.

Reply:
Hi @huwd I haven't seen the SLAs so if you do get some time to check in and revert that would be great! Thanks.